hasjob/views/index.py

```diff
@@ -537,6 +543,15 @@ def browse_tags():
     return render_template('tags.html.jinja2', tags=gettags(alltime=getbool(request.args.get('all'))))
 
 
+@app.route('/f/<name>', subdomain='<subdomain>', methods=['GET', 'POST'])
+@app.route('/f/<name>', methods=['GET', 'POST'])
+def filterset_view(name):
+    filterset = FilterSet.get(g.board, name)
+    return index(filters=filterset.to_filters(translate_geonameids=True),
```

Comment:
In what conditions would we have `translate_geonameids=False`?

Reply:
During the validation, `translate_geonameids` uses the default value of `False`.